tests/51media/30config.pl

```diff
@@ -0,0 +1,17 @@
+test "Can read configuration endpoint",
+   requires => [ $main::API_CLIENTS[0] ],
+
+   check => sub {
+        my ( $http ) = @_;
+        $http->do_request(
+            method   => "GET",
+            full_uri => "/_matrix/media/r0/config",
+        )->then( sub {
+            my ( $body, $response ) = @_;
+                
+            assert_json_keys( $body, qw( m.upload.size ) );
```

Comment:
given that this is optional, I'm not sure it is correct to assert it here.

You could assert that, if it is present, it is an int of a plausible size?